en/reference/annotations-mapping.rst

```diff
@@ -50,7 +50,9 @@ Optional attributes:
 
 -  **nodeType**: PHPCR type for this node, default ``nt:unstructured``.
 -  **repositoryClass**: Name of the repository to use for this document.
--  **versionable**: Set to true to enable versioning, implies *referenceable*.
+-  **versionable**: Set to ``"simple"`` or ``"full"`` to enable versioning 
+   (respectively simple or full level). Set to ``"false"`` to disable versioning
```

Comment:
The value for versionable attribute needs to be a string, not boolean. Its datatype is defined (in annotation mapping https://github.com/doctrine/phpcr-odm/blob/master/doctrine-phpcr-odm-mapping.xsd) as derived from ``xs:token``, which in turn is derived from ``xsd:normalizedString`` (see http://www.w3schools.com/schema/schema_dtypes_string.asp or http://books.xmlschemata.org/relaxng/ch19-77319.html).
If you try to set the value to boolean ``false``, instead of string ``"false"`` you get the following error:

*[Type Error] Attribute "versionable" of @PHPCR\Document declared on class <class> expects a(n) string, but got boolean.*

So I think that the description that I'm proposing is correct.